src/status_im/ui/screens/wallet/transactions/subs.cljs

```diff
@@ -74,11 +74,12 @@
          (group-by #(datetime/timestamp->date-key (:timestamp %)))
```

Comment:
Could you extract this whole logic in a function and add a test for it so that we don't introduce regression?